src/styles/blocks/_stars.scss

```diff
@@ -0,0 +1,25 @@
+@import "../utils/variables";
+
+.stars {
+  display: flex;
+  width: 166px;
+  flex-direction: row;
+  align-items: center;
+  position: relative;
+
+  &__star {
+    width: $Size-casual;
+    height: $Size-casual;
+    flex-shrink: 0;
+    position: relative;
+    background-image: url(../images/star.svg);
+    background-position: center;
+    background-repeat: no-repeat;
+    margin-right: 4px;
+  }
+
+  &--4 .stars__star:not(:nth-child(n+5)) {
+    background-image: url(../images/star-active.svg);
+  }
```

Comment:
Remember about working `stars--N` modifier! Same as it was in layout stars and product card